### DISCUSSION TITLE
SO offline

### DISCUSSION THREAD

**Discussion by WhiteNight121** [Security-Onion-Solutions/securityonion#2371] [Unsupported Versions] [answered]
Hello is it possible to run SO in an offline environment and if so is there a process (docs) to update it when its offline?

**Comment by idolaman** [comment, marked as answer]
It is possible to run SO in airgap mode.
You can update SO using soup command.
Here documentation about it:

[https://docs.securityonion.net/en/2.3/airgap.html#airgap](https://docs.securityonion.net/en/2.3/airgap.html#airgap)

**Reply by WhiteNight121** [reply]
Thank you so much
